guides/release/testing/index.md

```diff
@@ -16,18 +16,12 @@ You have a few options for running tests.
 
 First, you can run the test suite by entering the command `ember test`, or `ember t`, in your terminal. This will run the suite just once.
 
-Suppose, instead, you want the suite to run after every file change. You can enter `ember test --server`, or `ember t -s`.
```

Comment:
Why is this removing documentation for running `--server` flag of `ember test`? Shouldn't we keep the documentation but update it to `npm test -- --server`?

Reply:
it's easier to visit `/tests`.
You re-use the dev-server when poking around your app, too.

Reply:
I agree. But that's not covered by the referenced RFC, is it?